basepage/models.py

```diff
@@ -8,7 +9,7 @@ class Movie(models.Model):
     title = models.CharField(max_length=300)
     release_year = models.CharField(max_length=100)
     certificate = models.CharField(max_length=100)
-    runtime = models.CharField(max_length= 100)
+    runtime = models.CharField(max_length=100)
```

Comment:
Not sure why and how exactly there are 4 fields with start in their name. Could this value be held in a single field?